### PR TITLE
Update screenshot

### PR DESCRIPTION
**Pull request by davidszotten**
screenshot

### CODE REVIEW

**Comment by blueyed** [comment]
Thanks!

I think we should not have the binary screenshot in the repo, and also not the demo file.
It would be good to post the demo file here, as a comment / with the top comment, and upload the screenshot there also - and then use the URL for the image hosted by GitHub in the repo.
(It could also be hosted / maintained in a separate branch)

As for the demo/screenshot itself: it could feature some more things, e.g. `array.clear??` (for something that would have an actual (short) source then) - but it is not really clear to me.

I also have some WIP which allows for thread switching, but this could have its own demo screencast/GIF/screenshot then anyway.

File: screenshot/demo.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            array[i + 1] = array[i]
+            i = i - 1
+        array[i + 1] = key
+
+
+if __name__ == "__main__":
+
+    def canonic(self, filename):
+        # fake filename to match original screenshot, as an homage to the original author
+        return '/home/antocuni/env/src/pdb/demo.py'
```
Comment (blueyed) [inline screenshot/demo.py]: Nice idea, but not really necessary I think.

**Comment by davidszotten** [comment]
something like this?

File: screenshot/demo.py (hunk 1 above)
Comment (antocuni) [reply screenshot/demo.py]: thank you for the nice idea but I agree it's not really necessary :)

**Comment by blueyed** [comment]
> something like this?

Looks great, thanks! :)

**Comment by davidszotten** [comment]
created with 

```
def insertion_sort(array):
    breakpoint()
    i = 0
    j = 0
    n = len(array)
    for j in range(n):
        key = array[j]
        i = j - 1
        while i >= 0 and array[i] > key:
            array[i + 1] = array[i]
            i = i - 1
        array[i + 1] = key


if __name__ == "__main__":

    def canonic(self, filename):
        # fake filename to match original screenshot as an homage to the
        # original author
        return '/home/antocuni/env/src/pdb/demo.py'

    import pdb
    pdb.Pdb.canonic = canonic

    insertion_sort([3, 1, 2])
```

screenshot faked by combining multiple images, since the completions disappear when "used" or dismissed

**Pull request merged**